sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/Point.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The Point model.
+ */
+@Immutable
+public class Point {
+
+    /*
+     * The x-axis point coordinate.
+     */
+    private final Float xCoordinate;
+
+    /*
+     * The y-axis point coordinate.
+     */
+    private final Float yCoordinate;
+
+    /**
+     * Creates a Point object.
+     *
+     * @param xCoordinate The x-axis point coordinate.
+     * @param yCoordinate The y-axis point coordinate.
+     */
+    public Point(final Float xCoordinate, final Float yCoordinate) {
+        this.xCoordinate = xCoordinate;
+        this.yCoordinate = yCoordinate;
```

Comment:
These use the nullable `Float`, it is expected for these to be returned as `null`?

Reply:
Updated to primitive `float` as it cannot be null.